packages/zoe/src/contracts/sellItems.js

```diff
@@ -83,16 +97,11 @@ const start = zcf => {
 
     // The buyer's offer has been processed.
     buyerSeat.exit();
-    return defaultAcceptanceMsg;
-  };
 
-  /** @type {SellItemsPublicFacet} */
-  const publicFacet = {
-    getAvailableItems: () => {
-      assert(sellerSeat && !sellerSeat.hasExited(), `no items are for sale`);
-      return sellerSeat.getAmountAllocated('Items');
-    },
-    getItemsIssuer: () => issuers.Items,
+    if (publicFacet.getAvailableItems().value.length === 0) {
```

Comment:
we really should be using `math.isEmpty()` here. Hardcoding this sounds like a bad thing to teach people.

Reply:
done.